src/frontend/src/views/Upload/components/Metadata/components/StaticTable/components/Row/index.tsx

```diff
@@ -26,21 +34,53 @@ export default memo(function Row({ id, metadata }: Props): JSX.Element {
     publicId,
   } = metadata;
 
+  const validatedCellData: Record<
+    string,
+    React.ReactElement | string | undefined
+  > = {
+    privateId: privateId || "--",
```

Comment:
Nit: We have UNDEFINED_TEXT (`frontend/src/common/components/library/data_table/index.tsx`) and NO_CONTENT_FALLBACK (`frontend/src/common/components/library/Table/components/Cell/index.tsx`) doing same. Wonder if it's worth using one of those, although it already looks a bit scattered in our code base for standardizing on what we display when there's nothing to display.